modules/BookCollection.js

```diff
@@ -0,0 +1,27 @@
+export default class BookCollection {
+  #books = [];
+
+  constructor() {
+    this.#books = JSON.parse(localStorage.getItem('bookCollection')) || [];
+  }
+
+  addBook(book) {
```

Comment:
- [ ] Kindly consider taking ES6 arrow function here and in the other three instances of functions in this class as using the ES6 arrow function syntax is one of the requirements of this milestone.